velox/exec/HashTable.h

```diff
@@ -23,6 +23,11 @@
 
 namespace facebook::velox::exec {
 
+constexpr std::string_view kCapacity{"hashtable.capacity"};
```

Comment:
> kCapacity

These names cannot appear as is in velox::exec scope. There is simply not enough context in these: velox::exec::kCapacity.

Perhaps, move these inside the HashTable class?

Reply:
Sounds good. It would avoid copying the string.